### PR TITLE
Add absolute number to number range

### PR DESCRIPTION
**Pull request by ruthvik-17**
added some more edge case compatibility.

### CODE REVIEW

File: ner_v2/detectors/numeral/number_range/standard_number_range_detector.py [hunk 1]
```diff
@@ -245,10 +245,16 @@ def _get_number_range(self, min_part_match, max_part_match, full_match):
...
-            if int(entity_value_min) > int(entity_value_max):
-                temp = entity_value_max
-                entity_value_max = entity_value_min
-                entity_value_min = temp
+            try:
+                if int(entity_value_min) > int(entity_value_max):
+                    temp = entity_value_max
+                    entity_value_max = entity_value_min
+                    entity_value_min = temp
+            except :
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number_range/standard_number_range_detector.py]: E203 whitespace before ':'

**Comment by srajdev** [comment]
:+1: No lint errors found.

File: ner_v2/detectors/numeral/number_range/standard_number_range_detector.py [hunk 2]
```diff
@@ -245,10 +245,16 @@ def _get_number_range(self, min_part_match, max_part_match, full_match):
...
-                temp = entity_value_max
-                entity_value_max = entity_value_min
-                entity_value_min = temp
+            try:
+                if int(entity_value_min) > int(entity_value_max):
+                    temp = entity_value_max
+                    entity_value_max = entity_value_min
+                    entity_value_min = temp
+            except:
+                if float(entity_value_min) > float(entity_value_max):
```
Comment (amansrivastava17) [inline ner_v2/detectors/numeral/number_range/standard_number_range_detector.py]: shouldn't you can just check float? without try catch

File: ner_v2/detectors/numeral/number/en/data/units.csv [hunk 3]
```diff
@@ -1,5 +1,5 @@
 unit_type,unit_value,unit_variants
-currency,rupees,rupees | rupee | rs | rupya | rupaya | rupaye | rupye | rupay | paisa | paise | inr | ₹
+currency,rupees,rupees | rupee | rs | rupya | rupaya | rupaye | rupye | rupay | paisa | paise | inr | re | ₹
```
Comment (amansrivastava17) [inline ner_v2/detectors/numeral/number/en/data/units.csv]: `re` should be the variant, can you give any example?

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**